open_spiel/python/algorithms/alpha_zero/alpha_zero.py

```diff
@@ -284,16 +299,24 @@ def evaluator(*, game, config, logger, queue):
     az_player = game_num % 2
     difficulty = (game_num // 2) % config.eval_levels
     max_simulations = int(config.max_simulations * (10 ** (difficulty / 2)))
-    bots = [
-        _init_bot(config, game, az_evaluator, True),
-        mcts.MCTSBot(
+    bots = [_init_bot(config, game, az_evaluator, True)]
+    if config.search_type.lower() == "minimax":
```

Comment:
Do you want to compare against the minimax bot? or keep mcts as the evaluation opponent. I've considered adding others, eg external gtp bots. Keeping mcts as the evaluation opponent would make the results more comparable to usual AZ.

Reply:
Yeah, that was kind of the aim, but yeah I guess evaluating vs MCTS is better. I will fix that.